packages/author-byline/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@hashicorp/author-byline",
+  "name": "@hashicorp/react-author-byline",
```

Comment:
Make naming consistent with other react-components.

Reply:
We might want to add a changelog for these just so they get re-published with the new names.

Reply:
@BRKalow  These haven't been published yet via a changeset. I've been building these out as sub-components of what is to come `<SocialProof />` which is a composition of these new components.

Reply:
Sweet! Carry on 😄